package/src/dep/resolver.rs

```diff
@@ -0,0 +1,69 @@
+use alloc::{collections::BTreeMap, sync::Arc};
+
+use assembly::Library;
+
+use super::Dependency;
+use crate::{Digest, Package};
+
+// DEPENDENCY RESOLUTION
+// ================================================================================================
+
+#[derive(Debug, Clone)]
+pub enum DependencyResolution {
+    Local(LocalResolution),
+    // Registry(...),
```

Comment:
nit: can we remove this? As suggested in a previous comment, we could add a docstring and mention that a future variant could be a registry

Reply:
Sure. Done in https://github.com/0xPolygonMiden/miden-vm/pull/1544/commits/43e0a83488f491dfbe21305dae257336aa287fd8